pkg/runtime/bulk_subscriber.go

```diff
@@ -226,12 +226,16 @@ func (a *DaprRuntime) createEnvelopeAndInvokeSubscriber(ctx context.Context, psm
 	msg *pubsub.BulkMessage, route TopicRouteElem, bulkResponses *[]pubsub.BulkSubscribeResponseEntry,
 	entryIDIndexMap *map[string]int, path string, policy resiliency.Runner,
 ) error {
-	id, _ := uuid.NewRandom()
+	var id string
+	idObj, err := uuid.NewRandom()
+	if err != nil {
+		id = idObj.String()
```

Comment:
Here, need to check the error. If the kernel has not enough entropy, err will be non-nil, and id will be nil. So we need to do a check or there can be a panic.

Reply:
Yeah, agreed and we send the bulk message ahead with blank id, which shouldn't be an issue, as it won't be used for any other purpose other than back-tracking if user wants.